detox/test/e2e/03.actions.test.js

```diff
@@ -167,7 +167,10 @@ describe('Actions', () => {
     await element(by.id('toggleScrollOverlays')).tap();
 
     await element(by.id('ScrollView161')).swipe('up', 'slow', NaN, 0.9, 0.95);
-    await expect(element(by.text('Text1'))).not.toBeVisible(1);
+    if (device.getPlatform() === 'ios') {
+      // TODO: investigate why this assertion fails on Android
+      await expect(element(by.text('Text1'))).not.toBeVisible(1);
+    }
```

Comment:
This was already TODOed here:
https://github.com/wix/Detox/blob/d95b5e7d35e8e587aa1faa27031a49be5c649a74/detox/test/e2e/03.actions.test.js#L147-L150

@jonathanmos / @d4vidi FYI

Reply:
Actually, it's related to a very recently reported issue we have, and Amit already answered it:
https://github.com/wix/Detox/issues/3105#issuecomment-986229563